Packs/Slack/Integrations/SlackV3/SlackV3.py

```diff
@@ -1416,7 +1417,7 @@ async def listen(client: SocketModeClient, req: SocketModeRequest):
     if req.envelope_id:
         response = SocketModeResponse(envelope_id=req.envelope_id)
         await client.send_socket_mode_response(response)
-    if req.retry_attempt:
+    if req.retry_attempt and IGNORE_RETRIES:
```

Comment:
in case IGNORE_RETRIES=False, and req.retry_attempt > 0, we should log that a retry was handled and the the attempt count.